docs/content.zh/docs/ops/metrics.md

```diff
@@ -1317,6 +1317,42 @@ Note that the metrics are only available via reporters.
       <td>Current size of upload queue. Queue items can be packed together and form a single upload.</td>
       <td>Gauge</td>
     </tr>
+    <tr>
+      <th rowspan="7"><strong>Task/Operator</strong></th>
+      <td>startedMaterialization</td>
+      <td>The number of started materialization.</td>
+      <td>Counter</td>
+    </tr>
+    <tr>
+      <td>completedMaterialization</td>
+      <td>The number of successfully completed materialization.</td>
+      <td>Counter</td>
+    </tr>
+    <tr>
+      <td>failedMaterialization</td>
+      <td>The number of failed materialization.</td>
+      <td>Counter</td>
+    </tr>
+    <tr>
+      <td>lastFullSizeOfMaterialization</td>
+      <td>The full size of the last materialization part (in bytes).</td>
```

Comment:
This is actually last reported checkpoint, not last materialization, right?
How about
"The full size of the materialization part of the last reported checkpoint (in bytes)."
?

ditto: other added metrics.